secure-mobile-push/smp-android-client/securemobilepush/build.gradle

```diff
@@ -5,14 +5,25 @@ android {
     buildToolsVersion "25.0.2"
 
     defaultConfig {
-        minSdkVersion 16
+        minSdkVersion 9
         targetSdkVersion 25
         versionCode 1
-        versionName "1.0"
+        versionName "1.0.0"
 
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+    }
 
+    defaultConfig {
+        buildConfigField "String", "BEGIN_INDEX", "\"BEGIN-MESSAGE\""
+        buildConfigField "String", "END_INDEX", "\"END-MESSAGE\""
+        buildConfigField "String[]", "SMS_SENDER_NUMBER",
```

Comment:
Fixed in commit: https://github.com/adorsys/secure-key-storage/commit/c543cd4e7898e1a8cc65474c087c0f69515c2a15